### PR TITLE
Hackathon page, upgrade tailwindcss from v2 to v3

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: Samhita Alla <aallasamhita@gmail.com>

* Created a Hackathon page
* Upgraded TailwindCSS from v2 to v3

https://user-images.githubusercontent.com/27777173/152205234-901b10d3-a0d4-414b-b3a2-bb6a1ae8e57f.mov

@cosmicBboy, @SandraGH5, need your help with polishing the content.

### CODE REVIEW

File: hackathon.html [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        Join us for a 4-week virtual hackathon starting Feb 7!
+      </p>
+
+      <p class="py-3">
+        ML applications are built by many, but production-ready ML applications
+        are delivered by a few. Do you have the urge to venture into the MLOps
+        space to get the hang of how ML-in-production works in real-time? Let
+        this hackathon help you attain the required skill set to be a know-all
+        ML geek. The theme is, as you may have guessed, building end-to-end ML
+        applications with Flyte as the MLOps base.
```
Comment (cosmicBboy) [inline hackathon.html]:
```suggestion
        These days anyone can build ML models, but only a few can deliver
        production-ready ML applications. Do you want to venture into the MLOps
        space and get the hang of deploying ML in production? Join us in
        this hackathon to attain the skills to become a full-stack ML engineer.
        The theme is, as you may have guessed, building end-to-end ML
        applications on top of Flyte as the MLOps platform.
```

File: hackathon.html [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+      </p>
+      <p class="py-3">
+        Your idea could be around any ML or Data application. Pick up retail use
+        cases, fraud detection, time-series forecasting problems, feature
+        engineering, or literally, any ML or data use case you think adds value
+        when made production-ready, and we are good to go!
```
Comment (cosmicBboy) [inline hackathon.html]:
```suggestion
        Your idea could be around any ML or Data application. Pick up retail use
        cases, fraud detection, time-series forecasting problems, natural language processing
        computer vision, or literally, any ML or data use case you think adds real-world value
        in production, and we're good to go!
```

File: hackathon.html [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+      </p>
+      <p class="py-3 font-extralight">
+        This hackathon provides a chance to contribute to open source,
+        interact with experienced MLOps professionals, and have fun building
+        applications in the process.
```
Comment (cosmicBboy) [inline hackathon.html]:
```suggestion
        This hackathon gives you a chance to contribute to open source software,
        interact with experienced MLOps professionals, have fun building
        applications, and win some prizes in the process!
```

File: hackathon.html [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+        <li>Debug locally on local streamlit server</li>
+        <li>Merge to master</li>
+        <li>Streamlit cloud auto-deploy</li>
+      </ul>
+      <p class="py-5 font-semibold">
+        The code will be reviewed on a regular basis, and the team will provide
+        you the feedback.
+      </p>
+      <p class="py-3 font-extralight">
+        After the acceptance of your application, you will:
```
Comment (cosmicBboy) [inline hackathon.html]:
```suggestion
        At the end of the hackathon, you'll be able to present your project at the MLOps Community meetup,
        the Flyte judges will assess each project according to the judged by the criteria [add link to above section].
        
        The top 3 winners will be invited to:
```

File: hackathon.html [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+      </p>
+      <ul class="list-disc list-inside font-light pl-5">
+        <li>Present the application at Flyte/MLOps meetup</li>
+        <li>Write a blog post</li>
+        <li>Give an MLOps community coffee session interview</li>
```
Comment (cosmicBboy) [inline hackathon.html]:
```suggestion
        <li>Present their projects at the Flyte meetup</li>
        <li>Write a blog post</li>
        <li>Give an MLOps community coffee session interview</li>
```

**Comment by samhita-alla** [comment]
Yet to finish: we'll have to add links to the "Join Hackathon" button and Flytelab template link.

File: hackathon.html [hunk 6]
```diff
@@ -0,0 +1,296 @@
...
+      </p>
+      <p class="font-black">Feb 14 - Mar 14, 2022</p>
+      <p class="p-3 text-xl font-extralight">
+        Come build MLOps and Data products, have some fun, and win exciting
+        prizes!
+      </p>
+      <button
+        class="bg-transparent hover:bg-purple-500 text-purple-700 font-semibold hover:text-white py-3 px-4 border-purple-500 border-2 hover:border-transparent rounded mt-2"
+      >
+        Join Hackathon
```
Comment (cosmicBboy) [inline hackathon.html]: let's add this link to the button: https://docs.google.com/forms/d/1OpnCAv2AGjDBYO5gKj3mJ0B3nrBArlfGeptNpMAlRjQ

**Comment by samhita-alla** [comment]
@cosmicBboy, 
* incorporated all your suggestions,
* renamed the page title to "Engineering Lab", and
* fixed the page load issue

Screenshot 2022-02-04 at 4 15 19 PM

**Pull request merged**